src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+enum FigureType {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
-
+export interface Figure {
+  color: string;
```

Comment:
Why type of color string?  Maybe, you should use the type Color that you created in a first line